### DISCUSSION TITLE
Blank terminal after "lyricstify start"

### DISCUSSION THREAD

**Discussion by vaiebhavpatil** [lyricstify/lyricstify#261] [General]
No lyrics is displayed even after following all the mentioned steps. I installed directly using npm and configured using client id and everything but even then after starting nothing happens.

https://github.com/lyricstify/lyricstify/assets/73548646/ca14aad3-e605-4e5e-9c41-4f9a27ca490b

What am I doing wrong? And I made sure the song has spotify lyrics available.

**Comment by vaiebhavpatil** [comment]
anyone?

**Comment by vaiebhavpatil** [comment]
hello?

**Comment by LoveIiei** [comment]
same for me, pipe doesn't work as well, I saw someone mentioning that the API server is down or something.
